strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/domain/ArtifactEntry.java

```diff
@@ -75,7 +75,7 @@ public void setRepositoryId(String repositoryId)
         this.repositoryId = repositoryId;
     }
 
-    public ArtifactCoordinates getArtifactCoordinates()
+    public AbstractArtifactCoordinates getArtifactCoordinates()
```

Comment:
why we need to change this?

Reply:
To get objectId [here](https://github.com/strongbox/strongbox/pull/987/commits/8ec7ebbfc69f722fd5df46b7d3f78bf952b18d7d#diff-ca2af8f423f4756b1737ad2178c1101dR158). `artifactCoordinates` is indeed `AbstractArtifactCoordinates ` field.

Reply:
rolled back